apps/web/src/pages/TutorProfile.tsx

```diff
@@ -39,14 +37,39 @@ const TutorProfile: React.FC = () => {
         </Link>
         <Typography element="subtitle-2" className="font-bold text-natural-950">
           {intl("tutors.title")} /{" "}
```

Comment:
The forward slash should not be shown if the user name is not ready yet.

```suggestion
          {intl("tutors.title")} 
```

![Screenshot from 2025-02-03 19-17-56](https://github.com/user-attachments/assets/8b9fa598-5505-448f-b285-d6c7b0c6ca1d)